package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "jellyfin-web",
+    "description": "",
+    "version": "10.0.1",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "Jellyfin Project",
+    "license": "ISC",
```

Comment:
What does `ISC` mean? Shouldn't it be `GPLv2`?

Reply:
Yes forgot to change.

Reply:
@EraYaN can you fix the license?